### PR TITLE
Redesign the home page

### PR DESCRIPTION
**Pull request by ajay7231**
Resolving issue#6

### CODE REVIEW

**Comment by ajay7231** [comment]
scrnli_6_30_2021_6-12-18 PM

Currently home-ui look like this

**Comment by mridul-netizen** [comment]
@ajay7231 Responsive h kya ye ?

File: src/app/pages/home/home.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="about-section">
+<!-- <div class="about-section">
```
Comment (mridul-netizen) [inline src/app/pages/home/home.component.html]: Remove commented out code

File: src/app/components/preloader/preloader.component.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+export class PreloaderComponent implements OnInit {
+
+  @ViewChild('spinner-wrapper') spinner: ElementRef;
+
+  constructor() {
+
+    window.addEventListener('load', () => {
+      let preloaderFadeOutTime = 500;
+      function hidePreloader() {
+        // var preloader = $('.spinner-wrapper');
```
Comment (mridul-netizen) [inline src/app/components/preloader/preloader.component.ts]: remove commented out code

File: src/app/components/preloader/preloader.component.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+          this.spinner.nativeElement.fadeOut(preloaderFadeOutTime);
+        }, 500);
+      }
+      hidePreloader();
+    });
+    
+   }
+
+  ngOnInit(): void {
+  }
```
Comment (mridul-netizen) [inline src/app/components/preloader/preloader.component.ts]: remove unused functions

File: src/app/components/preloader/preloader.component.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+  constructor() {
+
+    window.addEventListener('load', () => {
+      let preloaderFadeOutTime = 500;
+      function hidePreloader() {
+        // var preloader = $('.spinner-wrapper');
+        setTimeout(function () {
+          this.spinner.nativeElement.fadeOut(preloaderFadeOutTime);
+        }, 500);
```
Comment (mridul-netizen) [inline src/app/components/preloader/preloader.component.ts]: [optional[
```suggestion
        setTimeout(() => {
          this.spinner.nativeElement.fadeOut(preloaderFadeOutTime);
        }, 500);
```

File: src/app/components/preloader/preloader.component.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+		-ms-transform: scale(0);
+		transform: scale(0);
+	} 40% { 
+		-webkit-transform: scale(1.0);
+		-ms-transform: scale(1.0);
+		transform: scale(1.0);
+	}
+}
+
+</style>
```
Comment (mridul-netizen) [inline src/app/components/preloader/preloader.component.html]: leave a line at the end of file

**Review by mridul-netizen — APPROVED** [review]
LGTM Minor nits!

File: src/app/components/preloader/preloader.component.html (hunk 5 above)
Comment (ajay7231) [reply src/app/components/preloader/preloader.component.html]: image
Responsive small screen devices view!!

**Pull request merged**